### PR TITLE
Update to latest release

### PR DESCRIPTION
**Pull request by leigh123linux**
It would be cool to have this in rpmfusion.

### CODE REVIEW

File: deadbeef-mpris2-plugin.spec [hunk 1]
```diff
@@ -24,25 +22,29 @@ to be orphaned. The original plugin supported MPRISv1 AND MPRISv2. This plugin
...
-%setup -q -n deadbeef-%{version}
+%autosetup -n deadbeef-%{version}
 
 
 %build
-%configure
-make %{?_smp_mflags}
+%configure \
+ --disable-silent-rules \
+ --enable-static
```
Comment (Vascom) [inline deadbeef-mpris2-plugin.spec]: Why static enabled?

**Review by Vascom — COMMENTED** [review]
Why do this?

**Comment by leigh123linux** [comment]
Sorry for the typo.

**Comment by Vascom** [comment]
I will create Review Request if rpmfusion today.

**Comment by leigh123linux** [comment]
You can assign it to me.

**Comment by Vascom** [comment]
As you wish.

**Pull request merged**